core/embed/trezorhal/stm32f4/xdisplay/st-7789/display_fb.c

```diff
@@ -213,6 +215,9 @@ void display_refresh(void) {
     return;
   }
 
+  // Disable access to the frame buffer from the unprivileged code
+  mpu_set_unpriv_fb(NULL, 0);
```

Comment:
maybe we should be calling this in deinit functions too?

Reply:
Yes, we should. Fixed, see 929ef8e8cf0561c7aefeb339395f49264f6ebe0b